### PR TITLE
Update ironbank build with latest docker context

### PR DESCRIPTION
**Pull request by pugnascotia**
The file `download.json` is replaced by `hardening_manifest.yaml`, which
includes various pieces of information about the Iron Bank build.

### CODE REVIEW

File: distribution/docker/src/docker/iron_bank/hardening_manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - filename: "elasticsearch-${version}-linux-x86_64.tar.gz"
+    url: "<artifact_path>/elasticsearch-${version}-linux-x86_64.tar.gz"
+  - filename: "tini"
+    url: "https://github.com/krallin/tini/releases/download/v0.19.0/tini-amd64"
+    validation:
+      type: "sha256"
+      value: "93dcc18adc78c65a028a84799ecf8ad40c936fdfc5f2a57b1acda5a8117fa82c"
+
+# List of project maintainers
+maintainers:
```
Comment (mark-vieira) [inline distribution/docker/src/docker/iron_bank/hardening_manifest.yaml:49]: Should/could we use distro lists here?

Comment (pugnascotia) [reply distribution/docker/src/docker/iron_bank/hardening_manifest.yaml:49]: What would they be?

@nkammah any thoughts here?

**Comment by pugnascotia** [comment]
@elasticmachine update branch

File: distribution/docker/src/docker/iron_bank/hardening_manifest.yaml (hunk 1 above)
Comment (nkammah) [reply distribution/docker/src/docker/iron_bank/hardening_manifest.yaml:49]: I don't see any mention in the doc on using distro lists - the different examples I've looked at in other projects uses individuals. If you're ok with it, let's stick to individuals for now.

**Review by nkammah — APPROVED** [review]
left minor comments but lgtm.

File: distribution/docker/src/docker/iron_bank/hardening_manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  BASE_IMAGE: "redhat/ubi/ubi8"
+  BASE_TAG: "8.3"
+
+# Docker image labels
+labels:
+  org.opencontainers.image.title: "elasticsearch"
+  # Human-readable description of the software packaged in the image
+  org.opencontainers.image.description: "You know, for search."
+  # License(s) under which contained software is distributed
+  org.opencontainers.image.licenses: "${license}"
```
[the remark this answers is not in the document]
Comment (pugnascotia) [reply distribution/docker/src/docker/iron_bank/hardening_manifest.yaml:25]: It'll "Elastic-License".

**Comment by pugnascotia** [comment]
Backported to `7.x` in 867754ed0c2

**Pull request merged**